tracim/views/contents_api/html_document_controller.py

```diff
@@ -0,0 +1,197 @@
+# coding=utf-8
+import transaction
+from pyramid.config import Configurator
+
+from tracim.models.data import UserRoleInWorkspace
+
+try:  # Python 3.5+
+    from http import HTTPStatus
+except ImportError:
+    from http import client as HTTPStatus
+
+from tracim import TracimRequest
+from tracim.extensions import hapic
+from tracim.lib.core.content import ContentApi
+from tracim.views.controllers import Controller
+from tracim.views.core_api.schemas import HtmlDocumentContentSchema
+from tracim.views.core_api.schemas import HtmlDocumentRevisionSchema
+from tracim.views.core_api.schemas import SetContentStatusSchema
+from tracim.views.core_api.schemas import HtmlDocumentModifySchema
+from tracim.views.core_api.schemas import WorkspaceAndContentIdPathSchema
+from tracim.views.core_api.schemas import NoContentSchema
+from tracim.lib.utils.authorization import require_content_types
+from tracim.lib.utils.authorization import require_workspace_role
+from tracim.exceptions import WorkspaceNotFound, ContentTypeNotAllowed
+from tracim.exceptions import InsufficientUserWorkspaceRole
+from tracim.exceptions import NotAuthenticated
+from tracim.exceptions import AuthenticationFailed
+from tracim.models.contents import ContentTypeLegacy as ContentType
+from tracim.models.contents import html_documents_type
+from tracim.models.revision_protection import new_revision
+
+HTML_DOCUMENT_ENDPOINTS_TAG = 'HTML documents'
```

Comment:
rename `HTML_DOCUMENT_ENDPOINTS_TAG` to ~~something like `SWAGGER_TAG_FOR_HTML_DOCUMENT_ENDPOINTS` or~~ `SWAGGER_TAG__HTML_DOCUMENT_ENDPOINTS`

Reply:
If i do this, i should do for others endpoints like workspace,users,threads,...